web/yo/app/views/gene.html

```diff
@@ -63,6 +63,7 @@
           <p ng-show="signatureCheck(gene.summary_review)" style="float:right;font-size:15px;">Updated by {{gene.summary_review.get('updatedBy')}} at {{gene.summary_review.get('updateTime')}}</p>
           <i class="fa fa-check" ng-class="iconClass('accept', gene.summary_review)" ng-if="iconExist('accept', gene.summary_review)"  ng-click="accept(null, 'GENE_SUMMARY', null, null, null, null, gene.summary_review)"></i>
           <i class="fa fa-times" ng-class="iconClass('reject', gene.summary_review)" ng-if="iconExist('reject', gene.summary_review)"  ng-click="reject(null, 'GENE_SUMMARY', null, null, null, null, gene.summary_review)"></i>
+          <img src="images/loading.gif" ng-if="iconExist('loading', gene.summary_review)"/>
```

Comment:
No need to add loading.gif. We have similar implementation for Update Gene button. `<button ....><span ng-show="!!!docStatus.savedGene"><i class="fa fa-spinner fa-spin"></i></span>...</button>` in the gene.html